homeassistant/components/homematic/light.py

```diff
@@ -116,5 +137,7 @@ def _init_data_struct(self):
 
         if self.supported_features & SUPPORT_COLOR:
             self._data.update({"COLOR": None})
+        if self.supported_features & SUPPORT_COLOR_TEMP:
+            self._data.update({"LEVEL": None})
```

Comment:
This part is not needed because we have already a level there. Do you have test this code?

Reply:
is there a way to specify that there are two updates for LEVEL? The color state also uses LEVEL on a second channel.

Based on the reply I can update this. I think it is safe to remove so worst case it would not get updated if the color is changed directly in the CCU instead of using home assistant.